test/e2e/fixture/fixture.go

```diff
@@ -666,6 +668,33 @@ func EnsureCleanState(t *testing.T, opts ...TestOption) {
 	FailOnErr(Run("", "kubectl", "create", "ns", DeploymentNamespace()))
 	FailOnErr(Run("", "kubectl", "label", "ns", DeploymentNamespace(), TestingLabel+"=true"))
 
+	// delete old namespaces used by E2E tests
+	namespaces, err := KubeClientset.CoreV1().Namespaces().List(context.Background(), v1.ListOptions{})
+	CheckError(err)
+	for _, namespace := range namespaces.Items {
+		if strings.HasPrefix(namespace.Name, "e2e-test-") {
```

Comment:
nit: use const `E2ETestPrefix` instead of `"e2e-test-"`

Reply:
:+1:, done.